cmd/lotus/daemon.go

```diff
@@ -191,13 +195,23 @@ var DaemonCmd = &cli.Command{
 		}
 
 		chainfile := cctx.String("import-chain")
-		if chainfile != "" {
+		snapshot := cctx.String("checkpoint")
+		if chainfile != "" || snapshot != "" {
+			if chainfile != "" && snapshot != "" {
+				return fmt.Errorf("cannot specify both 'snapshot' and 'import-chain'")
```

Comment:
```suggestion
				return fmt.Errorf("cannot specify both 'checkpoint' and 'import-chain'")
```

Reply:
- `checkpoint` is a term that carries social consensus connotations. 
- i think snapshot is the right term for the exported state. (ie not all snapshots one may load should be thought of as checkpoints)
- maybe the daemon load flag should be `lotus daemon --load-state=snapshot.car` or `lotus daemon --load-snapshot=snapshot.car`